nixos/modules/services/misc/bepasty.nix

```diff
@@ -136,48 +128,50 @@ in
               extraLibs = [ bepasty gevent ];
             };
           in {
-            BEPASTY_CONFIG = "${server.workDir}/bepasty-${name}.conf";
+            BEPASTY_CONFIG = "/tmp/bepasty.conf";
```

Comment:
You cannot assume that /tmp/bepasty.conf doesn't exist yet.

Reply:
You seem to have removed `PrivateTmp= enabled`. If you add that again, it might work as you intended.

Reply:
No worries, `DynamicUser` also enables `PrivateTmp`